.storybook/react-intl.js

```diff
@@ -0,0 +1,50 @@
+import flatten from "flat";
+
+/**
+ * Copied from https://stackoverflow.com/questions/50940640/how-to-determine-if-jest-is-running-the-code-or-not
+ */
+function isRunningJest() {
+    return process.env.JEST_WORKER_ID !== undefined;
+}
+
+/** Locales supported in the storybook "Globe" dropdown menu. */
+const locales = ["en-US", "fr", "unknown"];
+
+/** List of packages that will have localization support in Storybook. */ 
+const packages = [
+  "location-field",
+  "trip-details",
+  "trip-form"
+];
+
+/** Messages for all packages AND locales above. */
+const messages = {};
+
+if (!isRunningJest()) {
+  // Populate messages if not running snapshots.
+  // (Message printouts would be unnecessary replicated in snapshots without that check.)
+  packages.forEach((pkg) => {
+    locales.forEach((locale) => {
```

Comment:
We're setting messages here, but we're also doing it here? https://github.com/opentripplanner/otp-ui/pull/355/files#diff-09d1644f1de8e7f7e06a9d8d0603adaca2c57c37b7c8abe25d324632793e469fR260 (not sure if that link worked -- in TripDetails at the bottom of the file)

I may be missing something -- could you possibly explain what's going on?

Reply:
In the `TripDetails` story, we demonstrate overriding the built-in messages with custom ones. I'll add a comment regarding that.

Reply:
Covered in https://github.com/opentripplanner/otp-ui/pull/355/commits/bc143e233eb07e6fc967154384e6c1ef61c1696c.